lib/smart_answer_flows/marriage-abroad/outcomes/_services_and_fees.govspeak.erb

```diff
@@ -32,9 +32,9 @@
     <%= render partial: 'link_to_consular_fees.govspeak.erb',
                locals: { calculator: calculator } %>
   <% end %>
+<% end %>
 
-  <% if calculator.services.any? && calculator.services_payment_partial_name.present? %>
-    <% partial = 'payment_information_partials/' + calculator.services_payment_partial_name + '.govspeak.erb' %>
-    <%= render partial: partial, locals: { calculator: calculator } %>
-  <% end %>
+<% if calculator.services.any? && calculator.services_payment_partial_name.present? %>
+  <% partial = 'payment_information_partials/' + calculator.services_payment_partial_name + '.govspeak.erb' %>
+  <%= render partial: partial, locals: { calculator: calculator } %>
```

Comment:
Minor: I'm probably missing something, but I do slightly wonder whether the idea of "countries with consular facilities" is actually the correct concept to use i.e. is it possible that the fact that we need to use the `marriage-abroad-services.yml` just a reflection that the list of "countries with consular facilities" has not been kept up-to-date or does it provide more complex functionality. I guess I worry slightly that the `marriage-abroad-services.yml` mechanism doesn't give any explanation for why the payment info should or should not be shown in various circumstances. I'm sorry - I'm not sure I've explained that very well. Happy to chat about it if that would help.